suricata.yaml.in

```diff
@@ -660,6 +660,56 @@ af-packet:
     #use-mmap: no
     #tpacket-v3: yes
 
+dpdk:
+  eal-params:
+    proc-type: primary
+    a: 0000:3b:00.0
+    l: 0-1
+
+  # DPDK capture support
+  # Currently queues should be assigned to cores in ratio in 1:1 ratio
+  interfaces:
+    - interface: 0000:3b:00.0 # PCIe address of the NIC port
+      threads: auto # or number of threads - auto takes all lcores, no thread entry takes all cores
+      rx-queues: auto # or number of queues - auto assigns each core 1 queue
+      tx-queues: auto # or number of queues - auto assigns each core 1 queue
```

Comment:
there is a disconnect here
1. In IDS mode the packets are not intended to be send out to any interface. Hence enabling TX queue as `auto 1` is incorrect
2.  In IPS mode, the TX value depends upon the mapped or copy-interface RX queues to avoid use of mutex or lock or rin buffer.

Please rework